vitrina/datasets/migrations/0014_auto_20250811_1756.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.25 on 2025-08-11 14:56
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('vitrina_orgs', '0005_delete_agent'),
+        ('vitrina_datasets', '0013_alter_dataset_catalog'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='dataset',
+            name='information_system_creator',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.PROTECT, related_name='information_system_creator', to='vitrina_orgs.organization', verbose_name='Informacinės sistemos Valdytojas'),
+        ),
+        migrations.AddField(
+            model_name='dataset',
+            name='information_system_publisher',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.PROTECT, related_name='information_system_publisher', to='vitrina_orgs.organization', verbose_name='Informacinės sistemos Tvarkytojas'),
+        ),
+    ]
```

Comment:
The verbose_name attribute in migrations are different than the ones on the model, e.g.:

Migration: `verbose_name='Informacinės sistemos Valdytojas'`
Model: `verbose_name=_("Informacinės sistemos valdytojas")`

Let's regenerate the migrations instead of changing them by hand to not miss these inaccuracies.